app/sell/success.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import Page from '@components/Page'
+import Divider from '@components/Divider'
+import { Text, Button } from 'react-native-paper'
+import { StyleSheet, View } from 'react-native'
+import { useNavigation } from 'expo-router'
+import { clearRouterStack } from '@utils'
+
+const Success = () => {
+    const navigation = useNavigation()
+
+    const returnHome = () => {
+        clearRouterStack('/', navigation)
+    }
+    return (
+        <Page header="Success!">
+            <View style={styles.success}>
+                <Text style={styles.text}>
+                    Thank you for using MealMatch. Your funds will be released
+                    in the next 2 hours if there are no buyer complaints. {'\n'}
```

Comment:
Let's change this wording to "Thank you for using MealMatch. If there are no buyer complaints in the next two hours, your funds will be released sometime after that."